llvm/test/Transforms/DeadStoreElimination/inter-procedural.ll

```diff
@@ -0,0 +1,159 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -aa-pipeline=basic-aa -passes=function-attrs,dse -enable-dse-initializes-attr-improvement -S | FileCheck %s
```

Comment:
is `aa-pipeline` necessary?

also this shouldn't be running function-attrs

Reply:
Oh, you are right. Removed the `aa-pipeline` and `function-attrs`.